example/spec/parallel/testAPI/collectionSpec.js

```diff
@@ -46,13 +52,40 @@ describe('Collections API', () => {
         bucket: config.bucket,
         key: recordCreatedKey,
       })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordCreatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Create');
+      expect(omit(message.record, ['createdAt', 'updatedAt'])).toEqual(collection);
+    }
+  });
+
+  it('updating a collection publishes a record to the collection reporting SNS topic', async () => {
+    if (beforeAllFailed) {
+      fail('beforeAll() failed');
+    } else {
+      await apiTestUtils.updateCollection({
```

Comment:
We should use `updateCollection` from the `api-client` package. The `apiTestUtils` in the integration-tests package will/should be ultimately deprecated

Reply:
We should also update the test to use `createCollection` from the `api-client`

Reply:
[`84b2d25` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/84b2d25af943ef4258b05f7b430694c83e51fa61)